src/app/components/ScoreTable.js

```diff
@@ -0,0 +1,57 @@
+import ef from '../utils/elementFactory';
+
+const ScoreTable = (scoreList) => {
+  const header = ef(
+    'h3',
+    { className: 'scores__header' },
+    ef('img', {
+      src: 'static/assets/ui/scoreTable.svg',
+      className: 'scores__header-icon',
+    }),
+    'Mode Ranking',
+  );
+
+  // classes for cells
+  const cellClass = 'scores__table-cell';
+  const boldCellClass = `${cellClass} ${cellClass}--bold`;
+
+  // generating cells with columns titles
+  const columnTitles = ['Place', 'Player', 'Score'];
+  const columnTitleCells = columnTitles.reduce((acc, title) => {
+    acc.push(ef('p', { className: boldCellClass }, title));
```

Comment:
It is a table, but it doesn't  contain table semantic (thead, tbody, etc.) . Was there any reason for it?

Reply:
Less tags, simpler implementation and probably better responsiveness. Table would be more semantically appropriate, but there is no so much data, so I think it's not a big sin 👼🏻